index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>James Kapella || Web Developer</title>
+    <link rel='stylesheet' href='normalize.css'>
+    <link rel='stylesheet' href='styles.css'>
+    <link href="https://fonts.googleapis.com/css?family=Oswald:200" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,300,400" rel="stylesheet">
+</head>
+<body>
+
+    <header>
+        <div class='container'>
+            <h1>JAMES KAPELLA</h1>
+            <h3>Full stack web developer</h3>
+            <img class='scrollingFingerImage pulsingImage' src='img/scrollFingerIcon.png' alt='Scroll-down icon'>
+            <section>
+                <h4>About me</h4>
+                <p>I’m a Full Stack Developer-in-training with The Mayden Academy who’s enthusiastic about both front-end design and back-end systems. I’m also a Certified Scrum Master with training in Agile methodologies. I’m fascinated by games and games design, and have been jamming and creating in my spare time for several years. Away from keyboards and screens I can also be found making loud noises with electric guitars, and stroking all the good dogs. </p>
+            </section>
+            <section>
+                <h4>Contact</h4>
+                <ul>
+                    <li>
+                        <a href='https://github.com/JKapella' target='_blank'>
+                            <img src='img/githubIcon.png' alt='a github icon'> Github
+                        </a>
+                    </li>
+                    <li>
+                        <a href='mailto:jvkapella@gmail.com' target='_blank'>
+                            <img src='img/emailIcon.png' alt='an email icon'> Email
+                        </a>
+                    </li>
+                    <li>
+                        <a href='http://www.linkedin.com/in/james-kapella-895001106' target='_blank'>
+                            <img src='img/linkedinIcon.png' alt='a linkedin icon'> LinkedIn
+                        </a>
+                    </li>
+                    <li>
+                        <a href='http://culturebosh.com' target='_blank'>
+                            <img src='img/gameIcon.png' alt='a games icon'> My Games
+                        </a>
+                    </li>
+                </ul>
+            </section>
+        </div>
+    </header>
+    <main>
+        <div class='container'>
```

Comment:
could the class of container be put on the main instead and the extra div removed?

Reply:
Again, not easily, as the 'main' element sets the relative position of the right half of the site, and the container adds some responsive margins for the content within that, if you reckon it's really ugly I can change it though...

Reply:
Also, (sorry, not good at explaining) the container does the same job not matter what resolution/device view you're using - whereas the main and header sections behave differently depending on the device.

I don't think that was a very good justification.